<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DetectionResultPanel</class>
 <widget class="QWidget" name="DetectionResultPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="resultLabel">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>120</y>
     <width>231</width>
     <height>241</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nextNNButton">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>410</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Next NN</string>
   </property>
  </widget>
  <widget class="QPushButton" name="prevImgButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>412</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Previous Image</string>
   </property>
  </widget>
  <widget class="QLabel" name="imageLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>241</width>
     <height>231</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="prevNNButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>410</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Previous NN</string>
   </property>
  </widget>
  <widget class="QLabel" name="imgName">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="nnName">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nextImgButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>410</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Next Image</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveResultsButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>442</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Save Results</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
